chai/chai.d.ts

```diff
@@ -25,6 +28,28 @@ declare module Chai {
         (target: any, message?: string): Assertion;
     }
 
+    interface ShouldAssertion {
+        equal(val1: any, val2: any, msg?: string): void;
+        Throw: ShouldThrow;
+        throw: ShouldThrow;
+        exist(val: any, msg?: string): void;
```

Comment:
Happy with that - will make the change ([source code](https://github.com/chaijs/chai/blob/2.3.0/lib/chai/interface/should.js#L71) again)